data/data-pipeline/data_pipeline/etl/sources/cdc_svi_index/etl.py

```diff
@@ -0,0 +1,134 @@
+import pandas as pd
+
+from data_pipeline.etl.base import ExtractTransformLoad
+from data_pipeline.utils import get_module_logger
+from data_pipeline.score import field_names
+from data_pipeline.config import settings
+
+
+logger = get_module_logger(__name__)
+
+
+class CDCSVIINDEX(ExtractTransformLoad):
+    """CDC SVI Index class ingests 2018 dataset located
+    here: https://www.atsdr.cdc.gov/placeandhealth/svi/index.html
+    Please see the README in this module for further details.
+    """
+
+    def __init__(self):
+        self.OUTPUT_PATH = self.DATA_PATH / "dataset" / "cdc_svi_index"
+
+        self.CDC_SVI_INDEX_S3_URL = (
+            settings.AWS_JUSTICE40_DATASOURCES_URL + "/SVI2018_US.zip"
+        )
```

Comment:
Thank you to @esfoobar-usds for always being willing to assist with making datasets programmatically accessible.

Reply:
As much as possible, we prefer to load directly from source. It seems like we can access this data directly from `https://svi.cdc.gov/Documents/Data/2018_SVI_Data/CSV/SVI2018_US.csv` – can we point to that?